yellowbrick/features/projection.py

```diff
@@ -61,7 +61,7 @@ class ProjectionVisualizer(DataVisualizer):
         If classes is None and a y value is passed to fit then the classes
         are selected from the target vector.
 
-    color : list or tuple of colors, default: None
+    colors : list or tuple of colors, default: None
```

Comment:
Right, so I changed this because `colors` makes way more sense than `color` in the new context, particularly because it really must be a list. This meant a lot of changes in a lot of places - please be alert for ways I may have broken things just by adding an s! Unfortunately, VSCode wasn't a whole lot of help here ...

Reply:
Thanks for noting this! It was also something I was noticing since I make a lot of use of the colorizing params during my Visual Analytics class and various talks. I'm totally at peace with the change in spite of the backwards incompatibility here since I agree that the naming makes more sense (the inconsistency in how we name our colorizing params across &mdash; and even within &mdash; visualizers is often a headache)! Perhaps we can just make a good, clear note about how we've changed the color params here (and in, for instance, #917) in our release notes, and  in our [FAQ](https://www.scikit-yb.org/en/develop/faq.html#how-can-i-change-the-color-of-a-yellowbrick-plot) &mdash; thoughts?

Reply:
Absolutely, I'll happily update the FAQ!